src/applications/appeals/995/components/ContestableIssuesWidget.jsx

```diff
@@ -237,7 +237,7 @@ const ContestableIssuesWidget = props => {
             className="add-new-issue vads-c-action-link--green"
             to={{
               pathname: '/add-issue',
-              search: `?index=${items.length},new=true`,
+              search: `?index=${items.length}`,
```

Comment:
how come `new=true` is no longer needed?

Reply:
It wasn't ever needed. I was trying to figure out an easy way to pass data between pages without saving it to the form data or to session storage